motr/client.h

```diff
@@ -1867,9 +1868,9 @@ void m0_client_layout_free(struct m0_client_layout *layout);
 
 /*
  * Following dummy values are sent to S3 right now
- * curr_digest is set to '0 if flag M0_PI_CALC_UNIT_ZERO is on.
+ * curr_context is set to '0 if flag M0_PI_CALC_UNIT_ZERO is on.
```

Comment:
"curr_context" is a pointer.
What does it mean, "set it to '0'"?

Reply:
Removed this set of comments. Not required now.